exchanges/request/request.go

```diff
@@ -340,7 +340,7 @@ func (r *Requester) DoRequest(req *http.Request, path string, body io.Reader, re
 		}
 
 		if httpDebug {
-			dump, err := httputil.DumpResponse(resp, true)
+			dump, err := httputil.DumpResponse(resp, false)
```

Comment:
If I had HTTPDebugging set to true and verbose false, then I wouldn't receive the response. I think it would be better still to set this to true and remove the verbose log instead.